README.md

```diff
@@ -11,3 +11,6 @@ See [AWS Plugin](http://docs.getcloudify.org/latest/plugins/aws/)
 
 # Requirements
 boto3 AWS Python Library version 1.9.57
+
+# Examples
+For more examples navigate to [cloudify blueprints examples github repository](https://github.com/cloudify-community/blueprint-examples/).
```

Comment:
Let's reword it to say:

```For official blueprint examples using this Cloudify plugin, please see [Cloudify Community Examples](...).```

How does that sound?

Reply:
great.
maybe  we should write: [Cloudify Community Blueprints Examples]